modules/geo-layers/src/tile-3d-layer/utils.ts

```diff
@@ -0,0 +1,53 @@
+import {Vector3} from '@math.gl/core';
+import {Ellipsoid} from '@math.gl/geospatial';
+import type {OrientedBoundingBox} from '@math.gl/culling';
+import type {Tile3D} from '@loaders.gl/tiles';
+
+const scratchPoint = new Vector3();
+
+// Tile3D instance is not extensible. This is just a POC. Move into loaders.gl
+export function wrapTile(tile: Tile3D): Tile3D {
+  const unsealedTile = Object.create(tile);
+  Object.defineProperty(unsealedTile, 'bbox', {
```

Comment:
Nits: It doesn't matter for this temporary code but 
- I assume it is too late to start typing out `boundingBox`. Wish the `bbox` usage had been nipped in the bud, now it is spreading throughout the codebase.
- generally I prefer methods over accessors.